spec/index.html

```diff
@@ -191,22 +197,230 @@ <h2>Introduction</h2>
     <a href="#sec-grammar" class="sectionRef">Turtle Grammar</a>
     and <a href="#sec-parsing" class="sectionRef">Parsing</a>.
   </p>
+
+<section id="typographical-conventions" class="informative">
+  <h3>Typographical conventions</h3>
+
+  <p>The following typographic conventions are used in this specification:</p>
+
+  <dl class="typography">
+    <dt><code>markup</code></dt><dd>
+      Markup (elements, attributes, properties),
+      machine processable values (string, characters, media types),
+      property name,
+      or a file name is in red-orange monospace font.</dd>
+    <dt><dfn data-no-xref="">definition</dfn></dt><dd>
+      A definition of a term, to be used elsewhere in this or other specifications,
+      is in bold and italics.</dd>
+    <dt><a data-lt="definition">definition reference</a></dt><dd>
+      A reference to a definition <em>in this document</em>
+      is underlined and is also an active link to the definition itself. </dd>
+    <dt><a href="">hyperlink</a></dt><dd>
+      A hyperlink is underlined and in blue.</dd>
+    <dt>[<a href="">reference</a>]</dt><dd>
+      A document reference (normative or informative) is enclosed in square brackets
+      and links to the references section.</dd>
+    <dt><a href="" class="cp-terminal"><code>:</code></a></dt><dd>
```

Comment:
This is a floating `:`. See preview.

Reply:
As with the previous examples, this is intended to show how a code point reference is rendered. Of course, it could use a different character, or something else entirely. But, if we move the table down to the grammar section, the other typographic examples might not be worth carrying through.